README.md

```diff
@@ -1 +1,411 @@
+
+# grant (activate) the permissions
+az ad app permission grant --id ${AZURE_CLIENT_ID} --api 00000002-0000-0000-c000-000000000000 --expires never
+```
+
+You might see an error similar to the following, but that is OK, the permissions should have gone through still:
+
+_Operation failed with status: 'Conflict'. Details: 409 Client Error: Conflict for url: https://graph.windows.net/e7985bc4-a3b3-4f37-b9d2-fa256023b1ae/oauth2PermissionGrants?api-version=1.6_
+
+Finally, you need to grant admin permissions on the Azure Active Directory to the service principal because it will need to create other service principals for your AKSCluster:
+
+```console
+# grant admin consent to the service princinpal you created
+az ad app permission admin-consent --id "${AZURE_CLIENT_ID}"
+```
+
+_Note: You might need Global Administrator role to Grant admin consent for Default Directory. Please contact the administrator of your Azure subscription. To check your role, go to Azure Active Directory -> Roles and administrators. You can find your role(s) by clicking on Your Role (Preview)_
+
+After these steps are completed, you should have the following file on your local filesystem:
+
+- crossplane-azure-provider-key.json
+
+#### Setup Azure ProviderConfig
+
+Before creating any resources, we need to create and configure an Azure cloud provider resource in Crossplane, which stores the cloud account information in it. All the requests from Crossplane to Azure Cloud will use the credentials attached to this provider resource. The following command assumes that you have a crossplane-azure-provider-key.json file that belongs to the account you’d like Crossplane to use.
+
+Now we’ll create our Secret that contains the credential and ProviderConfig resource that refers to that secret:
+
+```console
+kubectl create secret generic azure-account-creds -n upbound-system --from-file=credentials=./crossplane-azure-provider-key.json
+```
+
+```console
+cat > provider.yaml <<EOF
+---
+apiVersion: azure.crossplane.io/v1beta1
+kind: ProviderConfig
+metadata:
+  name: default
+  namespace: upbound-system
+spec:
+  credentials:
+    source: Secret
+    secretRef:
+      namespace: upbound-system
+      name: azure-account-creds
+      key: credentials
+EOF
+```
+
+Apply it to the cluster:
+```console
+kubectl apply -f "provider.yaml"
+```
+The output will look like the following:
+
+```shell
+provider.azure.crossplane.io/default created
+```
+
+Crossplane resources use the ProviderConfig named ```default``` if no specific ProviderConfig is specified, so this ProviderConfig will be the default for all Azure resources.
+
+### We are now ready to provision resources:
+
+#### Create Network Fabric
+
+The example network composition includes the creation of a Resource Group, Virtual Network, and Subnets:
+
+```console
+kubectl apply -f examples/network.yaml
+```
+
+Verify status:
+
+```console
+kubectl get claim
+kubectl get composite
+kubectl get managed
+```
+
```

Comment:
i was expecting to see the readme instructions have a command to create the cluster also, what do you think about adding that?

Reply:
This was the issue we chatted about last week. Another one that needs to be applied across at least the AWS platform-ref. IF we all agree, I will update across all that need this addressed.

Reply:
oh ha you're right - we don't have it explicitly stated to create the cluster in platform-ref-aws either.  sure, feel free to open an issue to do that across all the repos.  lots of clean up coming our away across all the repos, it's great to get some more focus on them more broadly!